pkg/shared/families/malware/yara/config/config.go

```diff
@@ -21,6 +21,7 @@ type Config struct {
 	BinaryPath      string              `yaml:"binary_path" mapstructure:"binary_path"`
 	CompiledRuleURL string              `yaml:"compiled_rule_url" mapstructure:"compiled_rule_url"`
 	RuleSources     []config.RuleSource `yaml:"rule_sources" mapstructure:"rule_sources"`
-	YaracPath       string              `yaml:"yarac_path" mapstructure:"yarac_path"`
-	CacheDir        string              `yaml:"cache_dir" mapstructure:"cache_dir"`
+	// TODO(chrisgacsal): YaracPath parameter seems to be unused so we might want to remove it
```

Comment:
It is used for compiling raw Yara rules if raw rule sources are set:
https://github.com/openclarity/vmclarity/blob/fecce5bba0827bc699522946bab6a5391e51b4e0/pkg/shared/families/malware/yara/yara.go#L188

Reply:
Fixed [89df0da](https://github.com/openclarity/vmclarity/pull/814/commits/89df0da889823ffe481a45a53aad484964d27894)](https://github.com/openclarity/vmclarity/pull/814/commits/89df0da889823ffe481a45a53aad484964d27894)